test/parallel/test-fs-utimes.js

```diff
@@ -164,7 +164,23 @@ runTest(new Date('1982-09-10 13:37'), new Date('1982-09-10 13:37'), function() {
 });
 
 
+if (common.isWindows) {
+  common.refreshTempDir
+  const path = `${common.tmpDir}/test-utimes-precision`;
+  fs.writeFileSync(path, '');
+
+  const truncate_mtime = 1713037251360;
```

Comment:
I think it would help with readability if you could add a comment explaining why these magic values are magic (I had to dig into the libuv sources, and I’m still not quite sure I got it right). ;)